### PR TITLE
Create citation.cff

### PR DESCRIPTION
**Pull request by denis-yuen**
https://ucsc-cgl.atlassian.net/browse/DOCK-1864

https://citation-file-format.github.io/ and https://github.blog/changelog/2021-07-29-enhanced-citation-support/ if curious

### CODE REVIEW

File: citation.cff [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    family-names: Olunwa
+    given-names: Nneka
+  -
+    affiliation: "UC Santa Cruz Genomics Institute"
+    family-names: Hansen
+    given-names: Richard
+  -
+    affiliation: "UC Santa Cruz Genomics Institute"
+    family-names: Sheets
+    given-names: Elizabeth
```
Comment (bethsheets) [inline citation.cff:55]: orcid: "https://orcid.org/0000-0002-6335-4952"

**Review by bethsheets — CHANGES_REQUESTED** [review]
I added my orcid - thanks!

**Review by garyluu — APPROVED** [review]

**Review by NatalieEO — APPROVED** [review]

**Review by bethsheets — APPROVED** [review]

File: citation.cff [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    given-names: Ash
+  -
+    affiliation: "Ontario Institute for Cancer Research"
+    family-names: Cullion
+    given-names: Kim
+  -
+    affiliation: "Data Sciences Platform, Broad Institute"
+    family-names: "O'Connor"
+    given-names: Brian
+    name-particle: D
```
Comment (coverbeck) [inline citation.cff:69]: I'd never heard of a name-particle before and I had to look it up. :) I assume D is his middle initial, and I don't think a middle initial is a name-particle, is it? No big deal in any case.

**Review by coverbeck — APPROVED** [review]

File: citation.cff (hunk 2 above)
Comment (denis-yuen) [reply citation.cff:69]: Not sure, the name particle seems to either be used for "van", "van der", or middle initials in practice https://github.com/search?q=name-particle&type=code but that may be the blind leading the blind. 

The GitHub cite widget doesn't obviously look wrong after merging, so I think I'll allow it for now.

**Pull request merged**